assets/assets.go

```diff
@@ -1,4 +1,4 @@
 //go:generate go get github.com/jteeuwen/go-bindata/go-bindata
-//go:generate go-bindata -o templates.go -pkg assets templates/...
+//go:generate go-bindata -o templates.go -pkg assets templates/... locales/...
```

Comment:
Very minor thing, but we may want to rename `templates.go` to `data.go` or something to make it clear that it isn't just templates anymore.

Reply:
Updated